catalog/pipeline/fbc-release/0.11/README.md

```diff
@@ -0,0 +1,78 @@
+# FBC Release Pipeline
+
+ Tekton release pipeline to interact with FBC Pipeline                       
+
+## Parameters
+
+| Name | Description | Optional | Default value |
```

Comment:
The descriptions here should match the descriptions in the yaml file. Doesn't seem to be the case currently